components/admin_console/schema_admin_settings.jsx

```diff
@@ -560,7 +562,10 @@ export default class SchemaAdminSettings extends React.Component {
                 key={this.props.schema.id + '_bool_' + setting.key}
             >
                 <div className='banner__content'>
-                    <span>{this.renderBanner(setting)}</span>
+                    <span>
+                        { setting.banner_type === 'warning' ? <i className='fa fa-warning banner__icon'/> : null}
```

Comment:
Can we use the `WarningIcon` component from `components/widgets/icons/fa_warning_icon.jsx`. makes more sense to try to maintain in one single point all the icons.

Reply:
Didn't know about that `WarningIcon` component. Changed 👍